<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PACalibration</class>
 <widget class="QWidget" name="PACalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="paAtten">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="paReportDiff">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Report Diff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paCap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture PA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deviceDisconnect">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>PA Atten</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviceIP">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="paReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>repoert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration PA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DEVICE IP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="paResultList">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>871</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deviceConnect">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="paDiffIdx">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="paResultTable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
